### PR TITLE
Changes to support js with the node repl.

### PR DESCRIPTION
**Pull request by bounceme**
If you think I need to make any changes please let me know,thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Run `:Repl` without saving the code on a file.
 * Haskell
 * Python (You can chose python2 or python3 interpreter)
 * Erlang
+* Javascript
```
Comment (ujihisa) [inline README.md]: `s/s/S/`

**Comment by ujihisa** [comment]
Looks good! @aiya000 can you also check, and merge if it looks good to you?

**Comment by aiya000** [comment]
Yes !
I checked this. so good!!

**Comment by aiya000** [comment]
Thanks !! :sushi:

**Pull request merged**